opm/polymer/fullyimplicit/FullyImplicitCompressiblePolymerSolver.hpp

```diff
@@ -80,11 +80,23 @@ namespace Opm {
                 : rq(num_phases)
                 , rsSat(ADB::null())
                 , rvSat(ADB::null())
+                , fip()
             {
             }
+
+            enum FipId {
+                FIP_AQUA = BlackoilPropsAdInterface::Water,
+                FIP_LIQUID = BlackoilPropsAdInterface::Oil,
+                FIP_VAPOUR = BlackoilPropsAdInterface::Gas,
+                FIP_DISSOLVED_GAS = 3,
+                FIP_VAPORIZED_OIL = 4,
+                FIP_PV = 5,                    //< Pore volume
+                FIP_WEIGHTED_PRESSURE = 6
+            };
```

Comment:
The could/should perhaps be refactored to remove the duplication. 

This enum is used for indexing into a vector to avoid hard-coded (less readable) ints in the implementation.

I'm not sure what you mean by auto-generating the rest of the enum values. If you mean simply removing the "= 4" etc., that's simply a matter of taste in my opinion, and I'm happy either way